### PR TITLE
Message list and message details for campaigns.

### PR DESCRIPTION
**Pull request by peterp**

### CODE REVIEW

File: go/campaigns/templates/campaigns/incoming_detail.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+
+            </form>
+        </div>
+    </div>
+{% endblock %}
+
+{% block ondomready %}
+    {{ block.super }}
+    new go.campaign.bulkMessage.TextareaView({el: $('#frm-message textarea')});
```
Comment (justinvdm) [inline go/campaigns/templates/campaigns/incoming_detail.html:66]: I think we should put `TextareaView` in `go.components` or `go.components.<something>` instead of `go.campaign.bulkMessage`. This template seems unrelated to bulk messages, so `TextareaView` seems to be generally applicable. I think the same goes for `go.tables` (it should moved to `go.components.tables` or something). This stuff doesn't have to happen in this PR though.

Comment (peterp) [inline go/campaigns/templates/campaigns/incoming_detail.html:66]: Awesome! Good idea!

File: go/campaigns/templates/campaigns/incoming_detail.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
+{% extends "app.html" %}
+{% load url from future %}
+
+
+{% block content_extraclass %}campaigns{% endblock %}
+
+{# TODO, title #}
```
Comment (justinvdm) [inline go/campaigns/templates/campaigns/incoming_detail.html:7]: Why do we have a `TODO` for the title? Isn't the title `contact.msisdn`?

Comment (peterp) [inline go/campaigns/templates/campaigns/incoming_detail.html:7]: @smn will pipe in values, it's more for him to know that there's something here that he's potentially got to fill in.

File: go/campaigns/templates/campaigns/incoming_detail.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+{% block content_title %}
+    <a href="{% url 'campaigns:incoming_list' conversation.key %}">Messages</a>
+    &raquo;
+    {{ contact.msisdn }}
```
Comment (justinvdm) [inline go/campaigns/templates/campaigns/incoming_detail.html:11]: I don't think we are given `contact` when we render the `incoming_detail` view. 

@smn Does it make sense for this to always be `msisdn`, or should it depend on the conversation type or something?

File: go/campaigns/templates/campaigns/incoming_detail.html [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+                <table class="table">
+                    <thead>
+                        <tr>
+                            <th>Sender</th>
+                            <th>Messages</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {# TODO fill in messages. #}
+                        {% for message in message_list %}
```
Comment (justinvdm) [inline go/campaigns/templates/campaigns/incoming_detail.html:36]: I don't think we are given `message_list` when we render the `incoming_detail` view

File: go/campaigns/templates/campaigns/incoming_list.html [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+                        <th><input type="checkbox"></th>
+                        <th>Contact</th>
+                        <th>Messages Per Thread</th>
+                        <th>Messages</th>
+                        <th>Date</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {# TODO #}
+                    {% for message in message_list %}
```
Comment (justinvdm) [inline go/campaigns/templates/campaigns/incoming_list.html:53]: I don't think we are given `message_list` when we render the `incoming_list` view

File: go/campaigns/templates/campaigns/incoming_detail.html (hunk 4 above)
Comment (peterp) [inline go/campaigns/templates/campaigns/incoming_detail.html:36]: I made these up, @smn will come in and fill in the blanks when he's ready.

Comment (justinvdm) [inline go/campaigns/templates/campaigns/incoming_detail.html:36]: Ah ok cool.

File: go/campaigns/templates/campaigns/incoming_list.html (hunk 5 above)
Comment (justinvdm) [inline go/campaigns/templates/campaigns/incoming_list.html:53]: Ah, nvm. I suspect this is the same case as for `incoming_detail`

**Comment by justinvdm** [comment]
:+1: Its your call, but I think @smn should take a quick look so he is aware of the things he needs to fill in.

File: go/contacts/templates/contacts/group_detail.html [hunk 6]
```diff
@@ -2,6 +2,8 @@
 {% load url from future %}
 
 
+{% block content_extraclass %}campaigns{% endblock %}
```
Comment (justinvdm) [inline go/contacts/templates/contacts/group_detail.html:5]: Sorry to comment after +1ing, but just to check, does it make sense for `campaigns` to be here? It looks like it might not be related since this is a contacts template, not a campaigns template.

**Comment by peterp** [comment]
Hmpf. I think I forgot about this; @smn this was the messages stuff for G, I thought I merged it.
